Oleanparser.lean

```diff
@@ -50,7 +50,13 @@ def parseObj (objs : Std.HashMap UInt64 Obj) : ByteArrayParser Obj := do
     let utf8 ← readBytes size.toNat
     let utf8 := utf8.extract 0 (utf8.size - 1) -- drop zero terminator
     Obj.string <| String.fromUTF8Unchecked utf8 -- TODO
-  | 250 => error "mpz"
+  | 250 =>
+    let capacity ← read32LE
+    let signSize ← read32LE
+    let limbsPtr ← read64LE
+    -- 8 = sizeof(mp_limb_t) = sizeof(unsigned long int)  (except on Windows? 😱)
```

Comment:
I hope the except refers to `= sizeof(unsigned long int)` (which is *of course* 4 on windows..).  It would shatter my worldview if GMP used 32-bit limbs on 64-bit platforms.

Reply:
Oh right, I think I misremembered: the limb size is 64 bits, but the *API* uses `unsigned long`, so we had to resort to accessing the limb array directly in one or two places in Lean.